src/main/java/org/apache/sysds/parser/Statement.java

```diff
@@ -72,6 +72,9 @@
 	public static final String PS_MODE = "mode";
 	public static final String PS_GRADIENTS = "gradients";
 	public static final String PS_SEED = "seed";
+	public static final String PS_MODELAVG = "modelAvg";
+	public static final String PS_MODELS = "models";
```

Comment:
why do we need this besides modelAvg - remove if unnecessary.

Reply:
we need this beside modelAvg because it is one of the inputs of paramserver instruction.